src/pages/NftInfluencersTrx/Table/Table.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import cx from 'classnames'
+import { FluidSkeleton as Skeleton } from '../../../components/Skeleton'
+import Row from './Row/Row'
+import { useNftQuery } from '../../Index/Section/NftInfluencers/hooks'
+import styles from './Table.module.scss'
+
+export function formatKey({ trxHash, datetime }, idx) {
+  return `${trxHash}-${datetime}-${idx}`
+}
+
+const Table = ({ isMarket }) => {
+  const { data, loading } = useNftQuery(0, isMarket ? 5 : 25)
+
+  return (
+    <div className='column relative'>
+      {!isMarket && (
```

Comment:
To increase readability, move the market-related code to the standalone component